loghub/main.py

```diff
@@ -164,7 +166,7 @@ def format_changelog(repo,
         version=version, date=close_date, q=quotes)
 
     lines.append(header)
-    lines.append('### Bugs fixed\n')
+    lines.append('### Issues closed\n')
```

Comment:
The list is composed of Issues... 
and then there is another title that says

Pull requests... they are not on the same level, so I dont understand

>but this list is composed of Issues and PRs,